src/Templates/ProjectTemplates/CSharp/.NETCore/ConsoleApplication/NuGet.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0"
```

Comment:
We can revisit it. But the current approach we agreed to take is to get the existing functionality ported to MSBuild and then fix any issues with the existing design.

There are reasons for the "type: platform". Mostly because:

- How else are we supposed to know which dependency is the Shared Framework dependency?
- The existing design allows for other Shared Frameworks besides the Microsoft.NETCore.App.

/cc @davidfowl @anurse